stdlib/public/Platform/Platform.swift

```diff
@@ -109,6 +112,8 @@ public var errno : Int32 {
   set(val) {
 #if os(OSX) || os(iOS) || os(watchOS) || os(tvOS) || os(FreeBSD)
     return __error().pointee = val
+#elseif os(Windows)
+    return __errno().pointee = val
 #else
```

Comment:
This is becoming hard to read now, could you change `#else` to `#elseif os(Linux)`?  (Two spots in this file.)